src/HotChocolate/Spatial/src/Types/Extensions/SchemaBuilderExtensions.cs

```diff
@@ -0,0 +1,38 @@
+using HotChocolate.Execution.Configuration;
+using HotChocolate.Types.Spatial;
+using Microsoft.Extensions.DependencyInjection;
+using NetTopologySuite.Geometries;
+
+namespace HotChocolate
```

Comment:
The namespace is not correct.

`IRequestExecutorBuilder` extensions have to go into `Microsoft.Extensions.DependencyInjection`

`ISchemaBuilder ` extensions go into `HotChocolate`

DescriptorExtensions go into `HotChocolate.Types`

Also, classes that do go into `Microsoft.Extensions.DependencyInjection` need a unique name like

`HotChocolateSpatialRequestExecutorBuilderExtensions`

Reply:
in this case we need to split the class ....

Reply:
Resolved in #1870